kaztron/cog/dice.py

```diff
@@ -84,6 +86,25 @@ def __init__(self, bot):
         await self.bot.say("[{}]\n**Sum:** {:d}".format(' '.join(rolls_str), total))
         logger.info("Rolled FATE dice: {!r} (sum={})".format(rolls_str, total))
 
+    # strip_list = map(str.strip, lines)
+    @commands.command(pass_context=True, ignore_extra=False, no_pm=False)
+    @in_channels(ch_allowed_list)
+    async def choose(self, ctx, *, choices: str):
+        "Need some help making a decision? Just give the .choose command two or more options(separated by ,) and it will choose for you!"
```

Comment:
* Line over 100 chrs
* Doctypes conventionally use block strings:

```
"""
Docstring here
(and it's multiline!)
"""
```

* Docstrings must follow the standard convention elsewhere in kaztron (i.e. the argument and example sections)

Recommended:

```
"""
Need some help making a decision? Let the bot choose for you!

Arguments:
* choices - Two or more choices, separated by commas `,`.

Examples:
[give one or two examples here]
"""
```